src/encode.c

```diff
@@ -845,29 +851,41 @@ PyImaging_LibTiffEncoderNew(PyObject* self, PyObject* args)
 
     for (pos = 0; pos < d_size; pos++) {
         key = PyList_GetItem(keys, pos);
+        key_int = (int)PyInt_AsLong(key);
         value = PyList_GetItem(values, pos);
         status = 0;
-        TRACE(("Attempting to set key: %d\n", (int)PyInt_AsLong(key)));
+        core = 0;
```

Comment:
Is this used to indicate that this tag is a core tag, and not a custom one?

Perhaps something like `is_core_tag` will be clearer, at first glance the `if (core || !ImagingLibTiffMergeFieldInfo(&encoder->state, TIFF_DOUBLE, key_int))` below looked like it's checking for PIL imaging core.

Reply:
Fair enough. Done.